packages/createHash/src/index.ts

```diff
@@ -1,7 +1,13 @@
 import { isNodeLike } from "@apollo/utils.isnodelike";
 
 export function createHash(kind: string): import("crypto").Hash {
-  if (isNodeLike) {
+  // Some Node-like environments (like next.js Turbopack) apparently
+  // don't have module.require, so double-check before we call it.
+  // (But don't the value of isNodeLike because other logic depends on it,
+  // like Apollo Server signal handling defaults.) This does mean that
+  // Turbopack will call sha.js instead of the native crypto module, but
+  // it sure beats throwing because module.require does not exist.
+  if (isNodeLike && module.require) {
```

Comment:
I suppose one could be even more anal and do a typeof(module) check first but the error we're dealing with is about module.require not existing.